### PR TITLE
[INTEG] Dawn/Runtime proto synchronization

### PR DESCRIPTION
**Pull request by Alramech**

### CODE REVIEW

**Comment by brandonxxlee** [comment]
I made all relevant runtime changes on #197 and integration/chris. Why did we make a new separate branch?

**Comment by Alramech** [comment]
mostly because dawn got broken and I didn't want to go through the trouble of merging in a bunch of master changes, so I'm just manually picking the changes from those

File: ansible-protos/runtime.proto [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-        AUTO = 4;
-        ESTOP = 5;
-    }
+
     message ParamValue {
         string param = 1;
         oneof kind {
             float float_value = 2;
             int32 int_value = 3;
-            bool bool_value = 4;
```
Comment (brandonxxlee) [inline ansible-protos/runtime.proto]: This proto field should not be deleted.

File: runtime/testy/runtime.py [hunk 2]
```diff
@@ -54,7 +54,6 @@ def nonTestModePrint(*args):
     spawnProcess(PROCESS_NAMES.HIBIKE, startHibike)
     controlState = "idle"
     dawn_connected = False
-
```
Comment (brandonxxlee) [inline runtime/testy/runtime.py]: Unnecessary whitespace diff.

**Review by brandonxxlee — CHANGES_REQUESTED** [review]
Proto needs to be modified to keep up with changes on master.

File: ansible-protos/robotstate.proto [hunk 3]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+enum RobotState {
+     IDLE = 0;
+     TELEOP = 1;
+     AUTONOMOUS = 2;
+     ESTOP = 3;
+}
+
```
Comment (kevinji) [inline ansible-protos/robotstate.proto]: Extra newline here.

File: ansible-protos/runtime.proto [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
 message RuntimeData {
-    enum State {
-        STUDENT_CRASHED = 0;
-        STUDENT_RUNNING = 1;
-        STUDENT_STOPPED = 2;
-        TELEOP = 3;
-        AUTO = 4;
-        ESTOP = 5;
-    }
+
```
Comment (kevinji) [inline ansible-protos/runtime.proto]: Extra newline here.

**Review by kevinji — DISMISSED** [review]
Mostly whitespace changes. I'm not too sure how protobuf works; is there a way to sync Dawn's enums in renderer/utils/utils.js with the protobuf enums?

**Review by brandonxxlee — APPROVED** [review]
LGTM, just make sure the correct compiled protos are also built and merged into master.

**Comment by brandonxxlee** [comment]
Master protos have been changed, need to pull master and rebuild proto

File: dawn/fake-runtime/FakeRuntime.js [hunk 5]
```diff
@@ -19,10 +19,12 @@ const listenSocket = dgram.createSocket({ type: 'udp4', reuseAddr: true });
 const interval = 1000; // in ms
 
 const robotState = {
-  IDLE: 0,
-  RUNNING: 1,
-  ESTOP: 5,
+    IDLE: 0,
```
Comment (kevinji) [inline dawn/fake-runtime/FakeRuntime.js]: Can you make this two-space indented?

**Review by kevinji — DISMISSED** [review]
Minor spacing change.

**Comment by nikitakit** [comment]
It looks like we missed the window for deploying these changes this year. I really wish we could get these changes in, but it doesn't look like that will be happening.

I'm closing this PR.

**Pull request closed without merging**